### PR TITLE
Add Status to Monitoring software

### PR DESCRIPTION
**Pull request by dani3l0**
Thank you for taking the time to work on a PR for Awesome-Sysadmin!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software)
- [ ] Software your are submitting is not your own, unless you have a healthy ecosystem with a few contributors (which aren't your sock puppet accounts).
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 80 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at [awesome-selfhosted](https://github.com/awesome-foss/awesome-sysadmin)
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-foss/awesome-sysadmin/issues) or [PRs](https://github.com/awesome-foss/awesome-sysadmin/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

--------------

Please take some time to answer the following questions as best you can:

- **Why is it awesome?**

Status is a lightweight system monitoring app written in Python having only one dependency. It provides a fair amount of hardware information like CPU, physical memory, network and storage. Reported to work flawlessly on most Intel and AMD PCs, and even on Raspberry Pis. This makes Status a perfect solution for small homeservers.

- **Have you used it? For how long?**

Yes, I have. I am currently using Status on my homeserver with some self-hosted services. It is running painlessly since November 2022.

- **Is this in a personal or professional setup?**

Since it's running on a small homeserver, it's a personal setup.

- **How many devices/users/services/... do you manage with it?**

Status only allows monitoring machine it is hosted on.

- **Biggest pros/cons compared to other solutions?**

**Pros:** Easy to use and provides fairly enough information about the host. Has little requirements to make it running.

**Cons:** Very advanced sysadmins might find it too basic and missing some information. No Windows/Mac support.

- **Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**

Excellent is to have lightweight system monitoring app with decent-looking & responsive UI. The main limitation is no per-core CPU usage and temperature (however it is present in API).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -533,6 +533,7 @@ _See also: [IT Asset Management]([Ralph](#it-asset-management))_
 - [Sensu](https://sensuapp.org/) - Monitoring framework.
 - [ServerStatus BotoX](https://github.com/BotoX/ServerStatus) - Display and monitor your servers statistics in a beatiful way.
 - [ServerStatus moejda](https://github.com/mojeda/ServerStatus) - Server Status website script, displays uptime (days), free RAM, free HDD
+- [Status](https://gitlab.com/dani3l0/status) - Simple and lightweight system monitoring tool with a pleasant web interface. ([Demo](https://status-ksk5.onrender.com/)) `MIT` `Python`
```
Comment (nodiscc) [inline README.md:536]:
```suggestion
- [Status](https://gitlab.com/dani3l0/status) - Simple and lightweight system monitoring tool for small homeservers with a pleasant web interface. ([Demo](https://status-ksk5.onrender.com/)) `MIT` `Python`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by dani3l0** [comment]
This description sounds much better.

**Review by nodiscc — CHANGES_REQUESTED** [review]
Hi @dani3l0, since https://github.com/awesome-foss/awesome-sysadmin/pull/459 we have new contribution guidelines in place, could you please take some time to answer the following questions as best you can? Thanks

- **Have you used it? For how long?**
- **Is this in a personal or professional setup?**
- **How many devices/users/services/... do you manage with it?**
- **Biggest pros/cons compared to other solutions?**

**Comment by dani3l0** [comment]
Updated!

**Comment by dani3l0** [comment]
Hey, just moved Status to GitHub and fixed merge conflicts. Was the latter necessary?

**Review by nodiscc — APPROVED** [review]

**Pull request merged**